GitUI/CommandsDialogs/RevisionDiff.cs

```diff
@@ -63,6 +65,56 @@ public void RefreshArtificial()
             }
         }
 
+        #region Hotkey commands
+
+        public const string HotkeySettingsName = "BrowseDiff";
```

Comment:
`public const` are evil ;) this string _can_ change, so it isn't a const
please change to `public static readonly`

Reply:
I followed the pattern for all HotkeySettingsName.
Can all names be changed in a separate PR instead?